tools/bbbsupervisor/bbbsupervisor.go

```diff
@@ -102,38 +112,41 @@ func main() {
 	version := flag.Bool("version", false, "return program version")
 	flag.Parse()
 
-	fmt.Println("bbbsupervisor version", versionNum)
+	fmt.Printf("bbbsupervisor version %v starting..\n", versionNum)
```

Comment:
The idea of this version printout was to have it on start (in the log or stdout) and also use it for `bbbsupervisor --version` (thus the Exit 0) immediately after. With "starting..." this no longer is phrased right. I'd prefer to keep it as before.

Reply:
Ah, I didn't think about the behavior when called with `-version`. Good point, fixed.